### PR TITLE
{bio}[foss/2021b] Bismark v0.23.1

### PR DESCRIPTION
**Pull request by sassy-crick**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by sassy-crick** [comment]
Dependencies:
- [ ] Requires #14550

**Comment by bedroge** [comment]
Test report by @bedroge
**FAILED**
Build succeeded for 0 out of 2 (1 easyconfigs in total)
bob-Latitude-5300 - Linux Ubuntu 20.04, x86_64, Intel(R) Core(TM) i7-8665U CPU @ 1.90GHz, Python 3.8.10
See https://gist.github.com/7eaa71e53132fa5e01989d35b41ad8e1 for a full test report.

File: easybuild/easyconfigs/b/Bismark/Bismark-0.23.1-foss-2021b.eb [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+sources = ['%(version)s.tar.gz']
+checksums = ['2ceb620cfc3f1c869f4bda656ab01c242f1da7ec01615b0d23b28c702147a337']
+
+dependencies = [
+    ('Perl', '5.34.0'),
+    ('Bowtie2', '2.4.4'),
+    ('SAMtools', '1.14'),
+]
+
+sanity_check_commands = [ 'bismark --help']
```
Comment (bedroge) [inline easybuild/easyconfigs/b/Bismark/Bismark-0.23.1-foss-2021b.eb:31]:
```suggestion
sanity_check_commands = ['bismark --help']
```

**Review by bedroge — DISMISSED** [review]

File: easybuild/easyconfigs/b/Bismark/Bismark-0.23.1-foss-2021b.eb (hunk 1 above)
Comment (sassy-crick) [reply easybuild/easyconfigs/b/Bismark/Bismark-0.23.1-foss-2021b.eb]: I need to pick up my computer glasses!
Thanks for spotting that, that slipped through on my side.

**Comment by lexming** [comment]
@boegelbot: please test @ generoso

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 3 out of 3 (1 easyconfigs in total)
node309.hydra.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/02fbeab29a18618b816ccc7b9de78524 for a full test report.

**Review by lexming — APPROVED** [review]
LGTM

**Comment by lexming** [comment]
Going in, thanks @sassy-crick !

**Pull request merged**